### ISSUE TITLE
Assets not copied when path contains a space

### ISSUE THREAD

**Issue by Springrbua** [angular/angular-cli#15058]
# 🐞 Bug report

### Command (mark with an `x`)
```
- [ ] new
- [X] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?
No

### Description
When an Angular CLI Project is located under a folder with a space in the name, the assets are missing in the production bundles.  
A common case is the `Programm Files`-Directory in windows systems.

## 🔬 Minimal Reproduction
1. Create a new Angular application under a folder with a space in the name.
2. Add some assets to the application
3. Execute `ng build --prod`

The `assets` folder is missing in the created bundle.

## 🔥 Exception or Error
Unfortunately there is no Exception/Error.

## 🌍 Your Environment
<pre><code>
Angular CLI: 8.0.3
Node: 12.4.0
OS: win32 x64
Angular: 8.0.2
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.3
@angular-devkit/build-angular     0.800.3
@angular-devkit/build-optimizer   0.800.3
@angular-devkit/build-webpack     0.800.3
@angular-devkit/core              8.0.4
@angular-devkit/schematics        8.0.4
@angular/cdk                      8.0.1
@angular/cli                      8.0.3
@angular/flex-layout              8.0.0-beta.26
@angular/material                 8.0.1
@angular/pwa                      0.800.4
@ngtools/webpack                  8.0.3
@schematics/angular               8.0.4
@schematics/update                0.800.3
rxjs                              6.5.2
typescript                        3.4.5
webpack                           4.30.0
</code></pre>

**Anything else relevant?**
There is an [existing issue](https://github.com/angular/angular-cli/issues/11123) related to this, but it has been closed by the OP, as he found a workaround by using a different folder.
This does not seem to be the right solution for this, as it can cause wrong production bundles without showing an error or even a warning. 
Ideally, the CLI should be able to handle folders with space in the name correctly.
If that's not possible, it should abort the build and show an error.

**Comment by alan-agius4** [comment]
Also related to https://github.com/angular/angular-cli/issues/15032

**Comment by Springrbua** [comment]
@alan-agius4 thanks for pointing me to that issue.
In my case the Folder did contain brackets too, so I need to verify if spaces really cause problems or if it was just the brackets.

**Comment by Springrbua** [comment]
I tryed to reproduce this issue with custom folders and it was harder then I thought.
First I created a folder named `Folder with space` in my `Documents`-Folder but I couldn't reproduce the bug.
Then I created a folder named `FolderWith(B)racket` in my `Documents`-Folder and still couldn't reproduce the bug.  
Then I created the folder `C:\Folder with space` and I still couldn't reproduce the bug.
The last thing I tryed was creating a folder `C:\FolderWith(B)racket` and this time the assets-folder was missing.
So it seems like the brackets are causing the bug, but only if they are in the name of the root folder.
As the issue is not related to spaces but brackets, I close this one in favor of #15032.
